src/webgpu/util/conversion.ts

```diff
@@ -54,6 +69,67 @@ export function float32ToFloatBits(
   return (sign << (exponentBits + fractionBits)) | (newBiasedExp << fractionBits) | newFraction;
 }
 
+// Three partial-precision floating-point numbers encoded into a single 32-bit value all
+// sharing the same 5-bit exponent (variant of s10e5, which is sign bit, 10-bit mantissa,
+// and 5-bit biased (15) exponent).
```

Comment:
I'm not sure what s10e5 does here since it isn't a format that's related (it isn't a compressed format).

Reply:
Documentation on rgb9e5 like
https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_shared_exponent.txt
https://docs.microsoft.com/en-us/windows/win32/api/dxgiformat/ne-dxgiformat-dxgi_format
refer to rgb9e5 as related to s10e5 in that it has the same number of mantissa and exponent bits. (Though it's critically different in that the implicit leading value is 0, instead of 1)